controllers/nginx/pkg/template/template.go

```diff
@@ -342,8 +342,7 @@ var (
 
 func buildWhitelistVariable(s string) string {
 	if _, ok := whitelistVarMap[s]; !ok {
-		str := base64.URLEncoding.EncodeToString([]byte(s))
-		whitelistVarMap[s] = strings.Replace(str, "=", "", -1)
+		whitelistVarMap[s] = fmt.Sprintf("whitelist_%v", uuid.New())
```

Comment:
I only get the error with the rate limit whitelist, not `ingress.kubernetes.io/whitelist-source-range`